internal/identity/identitymanager.go

```diff
@@ -206,44 +216,41 @@ func (im *identityManager) firstVerifierForIdentity(ctx context.Context, vType c
 	return &verifiers[0].VerifierRef, false, nil
 }
 
-// ResolveNodeOwnerSigningIdentity add the node owner identity into a message
-func (im *identityManager) ResolveNodeOwnerSigningIdentity(ctx context.Context, msgSignerRef *core.SignerRef) (err error) {
-	verifierRef, err := im.GetNodeOwnerBlockchainKey(ctx)
+// ResolveNodeOwnerSigningIdentity adds the node owner identity into a message
+func (im *identityManager) ResolveNodeOwnerSigningIdentity(ctx context.Context, namespace string, signerRef *core.SignerRef) (err error) {
+	verifierRef, err := im.GetNodeOwnerBlockchainKey(ctx, namespace)
 	if err != nil {
 		return err
 	}
-	identity, err := im.GetNodeOwnerOrg(ctx)
+	identity, err := im.GetNodeOwnerOrg(ctx, namespace)
 	if err != nil {
 		return err
 	}
-	msgSignerRef.Author = identity.DID
-	msgSignerRef.Key = verifierRef.Value
+	signerRef.Author = identity.DID
+	signerRef.Key = verifierRef.Value
 	return nil
 }
 
 // GetNodeOwnerBlockchainKey gets the blockchain key of the node owner, from the configuration
-func (im *identityManager) GetNodeOwnerBlockchainKey(ctx context.Context) (*core.VerifierRef, error) {
-	if im.nodeOwnerBlockchainKey != nil {
-		return im.nodeOwnerBlockchainKey, nil
+func (im *identityManager) GetNodeOwnerBlockchainKey(ctx context.Context, namespace string) (*core.VerifierRef, error) {
+	if key, ok := im.nodeOwnerBlockchainKey[namespace]; ok {
+		return key, nil
 	}
 
-	orgKey := config.GetString(coreconfig.OrgKey)
-	if orgKey == "" {
-		orgKey = config.GetString(coreconfig.OrgIdentityDeprecated)
-		if orgKey != "" {
-			log.L(ctx).Warnf("The %s config key has been deprecated. Please use %s instead", coreconfig.OrgIdentityDeprecated, coreconfig.OrgKey)
+	defaultKey := im.namespace.GetDefaultKey(namespace)
```

Comment:
I think it's important that we do *not* use the `defaultKey` in the case of a root org registration.

e.g. when I have an NS config like this:

```yaml
  defaultKey: 0xabcd
  multiparty:
    enabled: true
    org:
      name: bob
      key: 0xbbb
```

Then my organization must be registered as `0xbbb` *not* `0xaaa`

Reply:
Alright, pushed another commit to differentiate and ensure that the "register org" path only looks at the `multiparty.org.key`, but all other paths should let `defaultKey` take precedence.